instrumentation/github.com/gocql/gocql/instrument.go

```diff
@@ -0,0 +1,110 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gocql
+
+import (
+	"log"
+
+	"go.opentelemetry.io/otel/api/global"
+
+	"go.opentelemetry.io/otel/api/metric"
+	"go.opentelemetry.io/otel/api/unit"
+)
+
+var (
+	// iQueryCount is the number of queries executed.
+	iQueryCount metric.Int64Counter
+
+	// iQueryErrors is the number of errors encountered when
+	// executing queries.
+	iQueryErrors metric.Int64Counter
```

Comment:
Do we need two instruments for these? Couldn't we add error labels to the count of a failed query?

Reply:
Thats a great idea!